recipes/gtlab-logging/all/conanfile.py

```diff
@@ -0,0 +1,109 @@
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools import files
+from conan.tools.build import check_min_cppstd
+from conan.errors import ConanInvalidConfiguration
+from conan import ConanFile
+import os
+
+required_conan_version = ">=1.59.0"
+
+class GTLabLoggingConan(ConanFile):
+    name = "gtlab-logging"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/dlr-gtlab/gt-logging"
+    topics = ("logging", "qt")
+    description = "Simple logging interface with qt support"
+
+    settings = "os", "arch", "compiler", "build_type"
+    
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_qt": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_qt": False
+    }
+
+    def requirements(self):
+        if self.options.with_qt:
+            self.requires("qt/6.7.0")
```

Comment:
```suggestion
            self.requires("qt/[>=5.15 <7]", transitive_headers=True)
```

@rainman110 After discussing about Qt with Conan team, we are accepting version range for same.

In case the user wants a specific version, the [replace_requires](https://docs.conan.io/2/reference/config_files/profiles.html#replace-requires) feature can be used to specify it.